.github/workflows/images.yml

```
@@ -0,0 +1,142 @@
+name: Images
+
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2 # https://github.com/marketplace/actions/checkout
```

Nit, I tend to prefer the slightly more compact version with the `-` on the same line as the key. Not blocking on this at all.
```suggestion
      - name: Checkout
        uses: actions/checkout@v2 # https://github.com/marketplace/actions/checkout
```

---

This was a copy from [the example](https://github.com/marketplace/actions/build-and-push-docker-images#git-context) and I inherited their weird YAML syntax 😄. I agree with you and I changed the dashes to be on the same line.